content-learn/mastering-service-meshes-for-developers/index.mdx

```diff
@@ -4,9 +4,9 @@ description: 'Learn how to build your own addons that will super charge your dev
 heroDescription: 'Addons give you superpowers to extend Storybook, automate workflows, and integrate with your favorite tools. This guide shows you how to create an addon.'
 overview: 'While the Storybook community offers over 250 addons, you can also build one tailored to your specific needs. This guide introduces you to the Addon Kit and APIs, along the way you will build an addon from scratch.'
 order: 4
-themeColor: '#EBC017'
+themeColor: '#292452'
```

Comment:
I wonder if we're able to source these from the central app.js / theme.js.

Reply:
Actually, these are not the brand colors but the colors taken from the service-mesh icon, which will be shown on the card.

Reply:
I am not sure, if this will be the best thing to do, else we can use the `tertiary color` that is the "Keppel Green" for the top-border.

Reply:
Oh, I see. I think the service mesh-specific colors are working (at least of what I've seen) and we should push forth this their use. If their brand color is referenced enough, you might consider centralizing them.

Reply:
Resolve conversation at your leisure. ✔️

Reply:
Sure, let's see how much more we use them in coming days then we can decide on that part.